Cargo.toml

```
@@ -41,13 +41,13 @@ cfg-if = "1.0"
 thiserror = "1.0"
 bzip2 = { version = "0.4.4", optional = true }
 flate2 = { version = "1.0", optional = true }
-liblzma = { version = "0.3.1", optional = true }
-bgzip = { version = "0.2.2", optional = true }
-zstd = { version = "0.12.2", optional = true }
+xz2 = { version = "0.1", optional = true }
+bgzip = { version = "0.3", optional = true }
+zstd = { version = "0.13", optional = true }
```

We don't want to revert #64 here =]
```suggestion
bgzip = { version = "0.3", optional = true, default-features = false, features = ["rust_backend"] }
liblzma = { version = "0.3.1", optional = true }
zstd = { version = "0.13", optional = true }
```

Also `bgzip` has new default features (`rayon` and `log`) that we don't quite need, so disabling default features and activating the one we actually need

---

God spot